public/sass/components/_tooltip.scss

```diff
@@ -102,7 +102,6 @@
   font-weight: 200;
   background-color: $tooltipBackground;
   border-radius: 5px;
-  z-index: 9999;
```

Comment:
this could affect other tooltips, not sure

Reply:
yes this is used by other tooltips so have to remain here.

Reply:
OK, I'll put it back and do some testing.  When I had left it there it didn't seem like the dynamic changes were taking effect

Reply:
I'm not sure why it wasn't taking effect before, but after putting it back the per element dynamic changes are working.